cmd/start.go

```diff
@@ -46,17 +48,19 @@ func init() {
 	startCmd.Flags().String("crd-version", "v1", "the version of the CRD you want monitored")
 	startCmd.Flags().String("crd-namespace", metav1.NamespaceNone, "(optional) the namespace of the CRD you want monitored, only needed for namespaced CRDs (ex: default)")
 	startCmd.Flags().String("templates", "", "absolute path to the directory with your template files")
-	startCmd.Flags().String("metrics-address", ":8080", "The address and port for the metrics endpoint")
-	startCmd.Flags().String("metrics-endpoint", "/metrics", "The URI for the metrics endpoint")
+	startCmd.Flags().String("server-address", ":8080", "The address and port for endpoints such as /metrics and /status")
+	startCmd.Flags().String("server-metrics-endpoint", "/metrics", "The URI for the metrics endpoint")
```

Comment:
This is really long for the command line. Here this can just be `metrics-endpoint` and below `status-endpoint`. Sorry for the confusion

Reply:
ahh, good call.